pkg/database/database.go

```diff
@@ -73,6 +75,10 @@ type Database struct {
 	statsCloser func()
 }
 
+func trim(s string) string {
```

Comment:
We can do it later, but I feel like this should be a more global thing that we use everywhere.

Reply:
yep - fixing the immediate known badness